Userland/Libraries/LibGUI/ScrollableContainerWidget.h

```diff
@@ -30,7 +30,7 @@ class ScrollableContainerWidget : public GUI::AbstractScrollableWidget {
     void update_widget_size();
     void update_widget_position();
     void update_widget_min_size();
-    virtual ErrorOr<void> load_from_gml_ast(NonnullRefPtr<GUI::GML::Node> ast, RefPtr<Core::Object> (*unregistered_child_handler)(DeprecatedString const&)) override;
+    virtual ErrorOr<void> load_from_gml_ast(NonnullRefPtr<GUI::GML::Node> ast, ErrorOr<NonnullRefPtr<Core::Object>> (*unregistered_child_handler)(DeprecatedString const&)) override;
```

Comment:
You don't need to do it here, but all these repeated `ErrorOr<NonnullRefPtr<Core::Object>> (*unregistered_child_handler)(DeprecatedString const&)` scream for a `using` alias

Reply:
Noted, I'll add that when I next work on this.